appeals/web/src/server/appeals/appeal-details/green-belt/__tests__/__snapshots__/green-belt.test.js.snap

```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`green-belt GET /change should render the greenBelt change page when accessed from Appellant Case page with a back link to appellant case page 1`] = `
```

Comment:
Just a thought, it's a little misleading that the name of the snapshot includes `with a back link` but the link isn't rendered in the snapshot. You could either render a snapshot that includes the whole page (although they do get quite large this way), or split the test so the snapshot can render just the back link (i.e. split the tests for `backLinkInnerHtml` and `mainInnerHtml`)